src/main/resources/anniv5Resources/localization/eng/siegepack/Keywordstrings.json

```
@@ -0,0 +1,17 @@
+[
+  {
+    "PROPER_NAME": "Blur",
+    "NAMES": [
+      "blur"
+    ],
+    "DESCRIPTION": "#yBlock is not removed at the start of your next turn."
```

```suggestion
    "DESCRIPTION": "#yBlock is not removed at the start of your turn."
```
It's not removed at the end of the current turn as well